lte/cloud/go/services/subscriberdb/digest_test.go

```diff
@@ -267,7 +267,7 @@ func TestGetPerSubscriberDigests(t *testing.T) {
 	// Initially the per-sub digest list should be empty (but not nil)
 	perSubDigests, err := subscriberdb.GetPerSubscriberDigests("n1")
 	assert.NoError(t, err)
-	assert.Empty(t, []*lte_protos.SubscriberDigestWithID{}, perSubDigests)
+	assert.Equal(t, []*lte_protos.SubscriberDigestWithID{}, perSubDigests)
```

Comment:
This is how I meant for all of these
```suggestion
	assert.Empty(t, perSubDigests)
```

Reply:
That makes sense, although in this specific instance I wanted to make sure that the `perSubDigests` is an empty slice instead of a `nil` slice, whereas `assert.Empty` will return `true` for either empty or nil slices. Therefore, I chose to use `assert.Equal` to make it clear that the slice needs to be empty, not `nil`. Wondering what's your thoughts on this? @hcgatewood

Reply:
I see. Okay so generally, the difference between a nil slice and an empty slice should be as nonexistent as possible [0] -- why do you want to ensure they're different here?

[0] Main reason, in my experience, where they differ is related to serialization. Is that what you're checking here, or ?

Reply:
The reason is that the return value of `GetPerSubscriberDigests`, i.e. the cloud-side per-subscriber digests, will be sent back to the agw in a `SyncSubscribersResponse`, and currently the agw is set to iterate over the list to update the agw per-sub digests. In python, iterating over `None` will result in an error, and that's why I wanted to enforce that the cloud side should always send back a non-nil array; should I stick to this pattern or try to add extra parsing steps on the agw side? @hcgatewood

Reply:
No that's a good reason. Just add a comment in the code at a relevant point to make it clear that's what you're doing